lib/cdo/developers_topic.rb

```diff
@@ -16,11 +16,16 @@ module DevelopersTopic
   # @return [String] The DOTD (without the '@' symbol), as per the Slack#developers topic.
   def self.dotd
     current_topic = Slack.get_topic 'developers'
-    raise 'developers topic not propertly formatted' unless current_topic.start_with? 'DOTD: @'
+    raise 'developers topic not propertly formatted' unless current_topic.start_with? 'DOTD: '
```

Comment:
Might as well make this a regex test. We expect something like
`DOTD: @?[a-z] DTS: ...`

Reply:
Done, and used the regex to grab the dotd's name too.